### DISCUSSION TITLE
Emails going to wrong email address

### DISCUSSION THREAD

**Discussion by hyena-gameworks** [emailjs-com/help#151] [Q&A]
Hello!

I am using emailjs to send emails as part of the contact form on my website. It was working perfectly at first, but after adding a new Gmail service (the old one was gmail too) so I can send emails to a different address, and after deleting the old service, emails still go to my old address. in the dashboard, only the new service is shown. When i click the service to see details, it shows the correct, new email address. I updated the service string on my frontend code and if I go to Email History, it correctly shows that the service ID for the recent emails matches the new service with the new email. however, emails still continue to be sent to my old email address. Any help would be deeply appreciated! Cheers

**Comment by xr0master** [comment]
The recipient's address is specified in the template.
